modyn/trainer_server/internal/trainer/pytorch_trainer.py

```diff
@@ -537,6 +596,14 @@ def train(self) -> None:  # pylint: disable=too-many-locals, too-many-branches
         self._log["num_batches"] = batch_number + 1
         self._log["total_train"] = total_stopw.measurements.get("TotalTrain", 0)
 
+        assert (
```

Comment:
Thanks, I think it's good to know for certain how many epochs we eventually trained

Reply:
cool, yeah, also to investigate/log the training samples vs time and investigate the weirdness we discussed today...